src/com/trilead/ssh2/packets/PacketKexDHInit.java

```diff
@@ -12,11 +10,11 @@
 {
 	byte[] payload;
 
-	BigInteger e;
+	byte[] publicKey;
 
-	public PacketKexDHInit(BigInteger e)
+	public PacketKexDHInit(byte[] publicKey)
```

Comment:
Im not sure how to convert to/from byte using BigInteger. I will just keep this class and create a new class with the changes.